lib/note-actions/index.tsx

```diff
@@ -0,0 +1,199 @@
+                className="theme-color-border"
+                checked={isPinned}
+                onChange={() => {
+                  this.pinNote(!isPinned);
+                }}
+              />
+            </span>
+          </label>
+
+          <label
+            className="note-actions-item"
+            htmlFor="note-actions-markdown-checkbox"
+          >
+            <span className="note-actions-item-text">
+              <span className="note-actions-name">Markdown</span>
+            </span>
+            <span className="note-actions-item-control">
+              <CheckboxControl
+                id="note-actions-markdown-checkbox"
+                checked={isMarkdown}
+                onChange={() => {
+                  this.markdownNote(!isMarkdown);
+                }}
+              />
+            </span>
+          </label>
+
+          <div className="note-actions-item note-actions-internal-link">
+            <div className="note-actions-copy">
+              {/* <p className="note-actions-detail note-actions-link-text theme-color-fg-dim">{`simplenote://note/${noteId}`}</p> */}
+              <ClipboardButton text={noteLink} linkText="Copy Internal Link" />
+            </div>
+          </div>
+
+          {hasRevisions && (
+            <div
+              className="note-actions-item"
+              onClick={this.props.toggleRevisions}
+            >
+              History…
+            </div>
+          )}
+          {hasRevisions || (
+            <div className="note-actions-item">History (unavailable)</div>
+          )}
+        </div>
+        <div className="note-actions-panel note-actions-public-link theme-color-border">
+          <label
+            className="note-actions-item"
+            htmlFor="note-actions-publish-checkbox"
+          >
+            <span className="note-actions-item-text">
+              <span className="note-actions-name">Publish</span>
+            </span>
+            <span className="note-actions-item-control">
+              <CheckboxControl
+                id="note-actions-publish-checkbox"
+                checked={isPublished}
+                onChange={() => {
+                  this.publishNote(!isPublished);
+                }}
+              />
+            </span>
+          </label>
+          <div className="note-actions-copy">
+            {isPublished && (
+              /* <p className="note-actions-detail note-actions-link-text theme-color-fg-dim">
+                {publishURL}
+              </p> */
+              <ClipboardButton text={publishURL} linkText="Copy Link" />
+            )}
+            {isPublished || (
+              <span className="note-actions-disabled">Copy Link</span>
+            )}
+          </div>
+        </div>
+        <div className="note-actions-panel theme-color-border">
+          <div className="note-actions-item" onClick={this.props.shareNote}>
+            Collaborate…
+          </div>
```

Comment:
Same thoughts here as my earlier comment. What do you think about changing this element to a `button`?